src/screens/Settings/components/WineSettings/index.tsx

```diff
@@ -307,7 +307,10 @@ export default function WineSettings({
             ])
             return toggleAutoInstallDxvk()
           }}
-          title={t('setting.autodxvk', 'Auto Install/Update DXVK on Prefix')}
+          title={t(
+            'setting.autodxvk',
+            'Auto Install/Update DXVK on Prefix (Improves performance for DirectX 9, 10 and 11 games)'
```

Comment:
Changing the key here won't make a difference since this is just the fallback in case the key is not translated for a specific language.
You need to change the english json file as well on the public folder.

Reply:
Understood.